doc/help/man-cupsd.conf.html

```diff
@@ -420,6 +420,16 @@ <h3 id="cupsd.conf-5.description.top-level-directives">Top-Level Directives</h3>
     <p style="margin-left: 2.5em; text-indent: -2.5em;"><strong>StrictConformance No</strong><br>
 Specifies whether the scheduler requires clients to strictly adhere to the IPP specifications.
 The default is &quot;No&quot;.
+</p>
```

Comment:
This needs to be in the cups-files.conf man page, not cupsd.conf... Also, you need to edit the man page source, not the HTML (which gets generated from the man page source...)

"man/cups-files.conf.5"

Reply:
Hi @michaelrsweet,

do we need to update `conf/cups-files.conf.in` as well?

Reply:
You could add it (commented out) to the cups-files.conf template, but it isn't required/necessary.